libs/application/templates/funding-government-projects/src/forms/FundingGovernmentProjectsForm.ts

```diff
@@ -123,6 +123,7 @@ export const FundingGovernmentProjectsForm: Form = buildForm({
             }),
             buildSliderField({
               id: 'project.refundableYears',
+              title: '',
```

Comment:
I know this is in line with other components, but we should just make title entirely optional instead of having to pass an empty string.

This applies to other components as well, could be a seperate PR

Reply:
Makes sense, I'll make a ticket